e2e/teams.loggedin.spec.ts

```diff
@@ -1,29 +1,35 @@
 import { Page, expect, test } from '@playwright/test'
 import { v4 as uuidv4 } from 'uuid'
-import { ADMIN_STORAGE_STATE, USER_STORAGE_STATE } from '../playwright.config'
+import {
+  USER_STORAGE_STATE,
+  USER_VALIDATED_STORAGE_STATE,
+} from '../playwright.config'
 import { getLocalStorage } from './helpers/localStorage'
 import {
   deleteTeamInvitationMessage,
   deleteTeamInviteAcceptanceMessage,
 } from './helpers/messages'
 import {
-  USER_NAME_LOCALSTORAGE_KEY,
   getAccessTokenFromCookie,
-  getAdminUserCredentials,
+  getAdminPAT,
   getUserIdFromLocalStorage,
   goToDashboard,
 } from './helpers/testUser'
+import {
+  USER_NAME_LOCALSTORAGE_KEY,
+  USER_VALIDATED_NAME_LOCALSTORAGE_KEY,
+} from './helpers/userConfig'
 
 const TEAM_NAME = 'swc-e2e-team-' + uuidv4()
-const INVITATION_MESSAGE = 'swc e2e test team invitation' + uuidv4()
+const INVITATION_MESSAGE = 'swc-e2e-invite'
 
```

Comment:
When running in headed mode, the original invitation text was too long to be fully displayed on the Teams page, so the row containing the invitation couldn't be found. Shorten invitation message so the row can be found.